operator/v1/types_network.go

```diff
@@ -234,8 +234,8 @@ type OpenShiftSDNConfig struct {
 	// +optional
 	MTU *uint32 `json:"mtu,omitempty"`
 
-	// useExternalOpenvswitch tells the operator not to install openvswitch, because
-	// it will be provided separately. If set, you must provide it yourself.
+	// useExternalOpenvswitch is ignored. Since OpenShift 4.7 the operator never
+	// deploys the openvswitch daemonset.
```

Comment:
```suggestion
	// useExternalOpenvswitch is ignored. This used to control whether Open vSwitch
	// was run as a system service or in a container, but as of OpenShift 4.7,
	// OpenShift always runs it as a system service.
```

Reply:
I modified this with your suggestion and updated the dependencies.